mlr-org/gallery/optimization/2023-01-23-shadow-variable-search-on-pima/index.qmd

```diff
@@ -0,0 +1,132 @@
+---
+title: "Shadow Variable Search on the Pima Indian Diabetes Data Set"
+description: |
+  Select features with a few lines of code.
+categories:
+  - feature selection
+  - classification
+author:
+  - name: Marc Becker
+    url: https://github.com/be-marc
+date: 2023-01-23
+bibliography: ../../bibliography.bib
+knitr:
+  opts_chunk:
+    R.options:
+      datatable.print.nrows: 6
+---
+
+{{< include ../_setup.qmd >}}
+
+```{r 2023-01-23-shadow-variable-search-on-pima-001}
+#| include: false
+
+lgr::get_logger("mlr3")$set_threshold("warn")
+lgr::get_logger("bbotk")$set_threshold("warn")
+set.seed(1)
+future::plan("multisession")
+```
+
+# Scope
+
+Feature selection is the process of finding an optimal set of features to improve the performance and robustness of machine learning algorithms.
```

Comment:
Hier vllt. noch kurz auf die unterschiedlichen Wege eingehen mit denen in mlr3 feature selection gemacht werden kann. Also filter vs wrapper.

Reply:
Und vor allem erklären was die generelle Idee von fselect ist, also dass man ML algos evaluiert und basierend auf den erbenissen neue features selektiert und am Ende dann normalerweise das beste erbenis nimmt.